stable/mysqldump/templates/configmap.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "mysqldump.fullname" . }}
+  labels:
+    app: {{ template "mysqldump.name" . }}
```

Comment:
Please make use of the standard labels, see https://docs.helm.sh/chart_best_practices/#standard-labels
(please apply to other templates as well)

Reply:
done